jib-core/src/main/java/com/google/cloud/tools/jib/api/JavaContainerBuilder.java

```diff
@@ -248,7 +235,8 @@ public JavaContainerBuilder addResources(Path resourceFilesDirectory) throws IOE
    */
   public JavaContainerBuilder addResources(Path resourceFilesDirectory, Predicate<Path> pathFilter)
       throws IOException {
-    return addDirectory(resourceFilesDirectory, resourcesPath, LayerType.RESOURCES, pathFilter);
+    classpath.add(RESOURCES_CLASSPATH);
```

Comment:
Look like this constant path will be added multiple times if this method is called multiple times. Doesn't seem necessary to have duplication?

Reply:
`classpath` is a `LinkedHashSet`, won't contain duplicates.